jekyll/_cci2/getting-help-with-2.0.md

```diff
@@ -1,11 +0,0 @@
----
-layout: classic-docs
-title: "Help and Support for 2.0"
-short-title: "Help and Support for 2.0"
-categories: [troubleshooting-and-debugging, help-and-support]
```

Comment:
We're violating our "No new 404s" mantra - you can use the `redirect:` variable in the front-matter as a temporary fix, value would likely be `/docs/2.0/help-and-support` (need to validate - typing by hand)